doc/api/vm.md

```diff
@@ -10,36 +10,73 @@ The `vm` module enables compiling and running code within V8 Virtual
 Machine contexts. **The `vm` module is not a security mechanism. Do
 not use it to run untrusted code**.
 
-JavaScript code can be compiled and run immediately or
-compiled, saved, and run later.
+JavaScript code can be compiled and run immediately or compiled, saved, and run
+later.
 
-A common use case is to run the code in a different V8 Context. This means
-invoked code has a different global object than the invoking code.
-
-One can provide the context by [_contextifying_][contextified] an
-object. The invoked code treats any property in the context like a
-global variable. Any changes to global variables caused by the invoked
-code are reflected in the context object.
+A common use case is to run code within a different environment. Each context
+uses a different V8 Context, meaning that it has a different global object than
+the rest of the code. One can provide a context by using the [`vm.Context`][]
+constructor.
 
 ```js
 const vm = require('vm');
 
 const x = 1;
 
-const context = { x: 2 };
-vm.createContext(context); // Contextify the object.
+const context = new vm.Context();
+context.global.x = 2;
```

Comment:
Would `context.globalThis` more appropriate to conform to the current ECMAScript spec?

Reply:
I think the exact name would be `globalObject`, but that would be confusing when paired with `globalThis`, so I thought it would be best to just call it `global`.